frontend/src/views/Abfrage.vue

```diff
@@ -344,7 +361,7 @@ export default class Abfrage extends Mixins(
     }
   }
 
-  async getAbfrageById(): Promise<void> {
+  async setSelectedAbfrageInStore(): Promise<void> {
     if (this.abfrageId !== undefined) {
       this.getInfrastrukturabfrageById(this.abfrageId, true)
         .then((dto) => {
```

Comment:
Alle `this.$store.commit("search/selectedAbfrage", XXX)` in dieser Methode können durch due Methode `saveAbfrageInStore` ersetzt werden.